docs/content/configuration/creating_volumestrorageclass.md

```diff
@@ -19,10 +19,10 @@ _<a name=spaceefficiency>**Table:**</a> `SpaceEfficiency` parameter definitions
 |Storage system type|SpaceEfficiency parameter options|
 |-------------------|---------------------------------|
 |IBM FlashSystem® A9000 and A9000R|Always includes deduplication and compression. No need to specify during configuration.|
-|IBM Spectrum® Virtualize Family|- `thick` (default value)<br />- `thin`<br />- `compressed`<br />- `deduplicated` <br /><br /> **Note:** If not specified, the default value is thick.|
-|IBM® DS8000® Family| - `none` (default value) <br />- `thin`<br /><br /> **Note:** If not specified, the default value is `none`.|
+|IBM Spectrum® Virtualize family|- `thick` (default value)<br />- `thin`<br />- `compressed`<br />- `dedup_thin` (creates volumes that are deduplicated with thin-provisioning)<br />- `dedup_compressed` (creates deduplicated and compressed volumes)<br />- `deduplicated` (creates deduplicated and compressed volumes, like `dedup_compressed`)<br /><br /> **Note:** <br />- The `deduplicated` value is deprecated. Use `dedup_compressed`, if possible.<br />- If not specified, the default value is `thick`.|
```

Comment:
could we also remove the space efficiency list duplication from [here](https://github.com/IBM/ibm-block-csi-driver/blob/8fca9f5ba4728154cc72731e26ba8e01737b6217/docs/content/using/sample_stateful_container.md)?
IMO we should maintain content duplications as few as possible

Reply:
updated in PR 415. I'm leaving the references in the using example as it's important and the user needs to note it specifically. certain things should be repeated in documentation

Reply:
as you can see from the miss here, and in general from many previous ones, it is not practical for us to maintain duplications of such specific details, as this exhaustive list.
no matter how important it is, we just don't have a way to keep track of the duplications.
in practice, it is more likely to cause false information, which is worst than omitting it.
please remove the specific values from here, at least until we find a practical way for keeping them correct

Reply:
DONE in PR [415](https://github.com/IBM/ibm-block-csi-driver/pull/415/)